### DISCUSSION TITLE
Are these equivalent!

### DISCUSSION THREAD

**Discussion by Abhi-SimuTech** [ansys/pyaedt#3672] [Q&A]
Hi,

What may be the differences (in output geometry) if I replace the UDM code section in (1) , with PyAEDT function in (2):

(1) ![Image_1](https://github.com/ansys/pyaedt/assets/145813888/54aa7f66-4363-474b-a96a-a8740f0dca70)

(2) ![Image_2](https://github.com/ansys/pyaedt/assets/145813888/c2732a84-a19b-4756-be0c-558952ffdcbb)

where,

![Image_3](https://github.com/ansys/pyaedt/assets/145813888/cf4a73bb-5e36-4db7-81ee-e6848a0e7d70)

**Comment by Samuelopez-ansys** [comment]
Hi @Abhi-SimuTech ,

Please, share snippet codes to reproduce the issue. We should be able to run both codes end-to-end to test the issue, if not is quite difficult to figure out what is happening. 

Thanks
